_maps/map_files/KiloStation/KiloStation.dmm

```diff
@@ -43364,6 +43364,12 @@
 /obj/structure/cable,
 /turf/open/floor/iron/white,
 /area/security/prison)
+"ehM" = (
+/mob/living/simple_animal/hostile/carp{
```

Comment:
if you could give this carp a different name or some other way of indicating that it's different from a normal carp, that'd be nice.

Reply:
Did that'!